tableau-databricks/connection-resolver.tdr

```diff
@@ -24,16 +24,15 @@ limitations under the License.
 	
         <connection-normalizer>
             <required-attributes>
-                <setImpersonateAttributes/>
                 <attribute-list>
                     <attr>class</attr>
                     <attr>server</attr>
-                    <attr>dbname</attr>
+                    <attr>v-http-path</attr>
                     <attr>one-time-sql</attr>
                     <attr>authentication</attr>
                     <attr>username</attr>
                     <attr>password</attr>
-                    <attr>authentication-type</attr>
```

Comment:
is this deprecated?

Reply:
This corresponds to the AuthMech parameter in the connection string. It's hard-coded to 3, so this field is unused.